diff --git a/dist/Socket/index.js b/dist/Socket/index.js
index eb1aa4dd806c29f86c309d00422b32fb526e2d95..6884a6f2024835f344b3878d08df60a5f1d29328 100644
--- a/dist/Socket/index.js
+++ b/dist/Socket/index.js
@@ -58,7 +58,7 @@ const sessions = new Map();
 const callback = new Map();
 const retryCount = new Map();
 const P = require("pino")({
-    level: "silent",
+    level: process.env.WA_DEBUG_SIGNAL === "1" ? "debug" : "silent",
 });
 const startSession = (...args_1) => __awaiter(void 0, [...args_1], void 0, function* (sessionId = "mysession", options = { printQR: true }) {
     if (isSessionExistAndRunning(sessionId))
@@ -68,7 +68,7 @@ const startSession = (...args_1) => __awaiter(void 0, [...args_1], void 0, funct
         const { state, saveCreds } = yield (0, baileys_1.useMultiFileAuthState)(path_1.default.resolve(Defaults_1.CREDENTIALS.DIR_NAME, sessionId + Defaults_1.CREDENTIALS.PREFIX));
         const sock = (0, baileys_1.default)({
             version,
-            auth: state,
+            auth: { creds: state.creds, keys: (0, baileys_1.makeCacheableSignalKeyStore)(state.keys, P) },
             logger: P,
             markOnlineOnConnect: false,
             browser: baileys_1.Browsers.ubuntu("Chrome"),
@@ -167,7 +167,7 @@ const startSessionWithPairingCode = (sessionId, options) => __awaiter(void 0, vo
         const sock = (0, baileys_1.default)({
             version,
             printQRInTerminal: false,
-            auth: state,
+            auth: { creds: state.creds, keys: (0, baileys_1.makeCacheableSignalKeyStore)(state.keys, P) },
             logger: P,
             markOnlineOnConnect: false,
             browser: baileys_1.Browsers.ubuntu("Chrome"),
